quaddtype/numpy_quaddtype/src/umath/unary_ops.cpp

```diff
@@ -156,6 +156,9 @@ init_quad_unary_ops(PyObject *numpy)
     if (create_quad_unary_ufunc<quad_absolute, ld_absolute>(numpy, "absolute") < 0) {
         return -1;
     }
+    if (create_quad_unary_ufunc<quad_absolute, ld_absolute>(numpy, "fabs") < 0) {
```

Comment:
Could you add a comment for why we alias - that we don't handle complex values - just in case we would support that in the future?